jinja2/filters.py

```diff
@@ -457,7 +457,9 @@ def do_indent(s, width=4, indentfirst=False):
             indent by two spaces and indent the first line too.
     """
     indention = u' ' * width
-    rv = (u'\n' + indention).join(s.splitlines())
+    lines = s.splitlines()
+    rv = lines.pop(0) + u'\n'
+    rv += u'\n'.join([indention + l if l else l for l in lines])
```

Comment:
lowercase-L is not a good choice (and actually disallowed as per PEP8, just like uppercase-i)